vhdl_lang/src/analysis/static_expression.rs

```diff
@@ -0,0 +1,478 @@
+use crate::analysis::static_expression::BitStringConversionError::EmptySignedExpansion;
+use crate::ast::{BaseSpecifier, BitString};
+use crate::Latin1String;
+use itertools::Itertools;
+use std::iter;
+
+macro_rules! byte_is_odd_decimal {
```

Comment:
This could just be a function, I typically only use macros when a function is not possible.